src/components/authorise/authorise-get-controller.ts

```diff
@@ -68,6 +79,16 @@ const handleRequestError = (
         `${error.redirectUri}?error=invalid_request&error_description=${encodeURIComponent("Invalid Request")}`
       );
     }
+  } else if (error instanceof ReturnCodeError) {
+    if (config.getReturnCode()) {
+      res.redirect(
+        `${error.redirectUri}?error=invalid_request&error_description=${encodeURIComponent(error.message)}`
+      );
```

Comment:
this is incorrect - we don't return an error at this point if return codes are on, we return an auth code as normal. Because we're not baking in any logic around return codes at this point, having a return code will not affect the response in the user info endpoint either, apart from including the return code